### PR TITLE
add contributing.md

### PR DESCRIPTION
**Pull request by luckykumarirai**
Add CONTRIBUTING.md file with all important guidelines and screenshots.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+## Any contributions you make will be under the MIT Software License
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's [issues](https://github.com/briandk/transcriptase-atom/issues)
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](); it's that easy!
```
Comment (Pranav016) [inline CONTRIBUTING.md:138]: Add this to the hyper link-
https://github.com/ALPHAVIO/BlogSite/issues

**Review by Pranav016 — CHANGES_REQUESTED** [review]
Great work, minor change required.

**Comment by luckykumarirai** [comment]
I have updated that link.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+## Any contributions you make will be under the MIT Software License
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's [issues](https://github.com/ALPHAVIO/BlogSite/issues)
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](); it's that easy!
```
Comment (Pranav016) [inline CONTRIBUTING.md:138]: Update -
```
[opening a new issue]()
```
To this-
```
[opening a new issue](https://docs.github.com/en/github/managing-your-work-on-github/creating-an-issue)
```

**Review by Pranav016 — CHANGES_REQUESTED** [review]
Also squash your commits since there will be 3 commits trying to do the same thing or you can cop the file, delete this branch and make a new PR with a single commit.
Other wise, Great Work 👏 

[Steps to squash for help](https://github.com/ALPHAVIO/ToDoist/pull/6)

**Pull request closed without merging**